custom_dc/build_image.md

```diff
@@ -41,84 +41,68 @@ If you want to pick up the latest prebuilt version, do the following:
    ```
 
 ## Build a local image {#build-repo}
-
 You will need to build a local image in any of the following cases:
 - You are making substantive changes to the website UI
 - You are ready to deploy your custom site to GCP
 
-Rather than building from "head", that is, the very latest changes in Github, which may not have been tested, we recommend that you use the tested "release" equivalent of the stable Docker image. This release uses the tag `customdc_stable`, and is available at [https://github.com/datacommonsorg/website/releases/tag/customdc_stable](https://github.com/datacommonsorg/website/releases/tag/customdc_stable){: target="_blank"}.
-
-> **Note:** If you are working on a large-scale customization, we recommend that you use a version control system to manage your code. We provide procedures for Github, and assume the following:
-- You have a Github account and project.
-- You have created a fork off the base Data Commons `website` repo (https://github.com/datacommonsorg/website){: target="_blank"} and a remote that points to it, and that you will push to that fork. 
-
-
-### Sync a local workspace to the stable release
+Rather than building from the master branch, which includes the very latest changes in Github, that may not have been tested, we recommend that you use the tested "stable" branch equivalent of the stable Docker image. This branch is `customdc_stable`, and is available at [https://github.com/datacommonsorg/website/tree/customdc_stable](https://github.com/datacommonsorg/website/tree/customdc_stable){: target="_blank"}.
```

Comment:
Optional minor grammar thing: "that may not have been tested" kind of makes this sentence not flow IMO. WDYT of splitting into two sentence? "Building from the master branch includes the very latest changes in Github, which may not have been tested. Instead, we recommend that you use..."